maps/models.py

```diff
@@ -457,6 +457,20 @@ class Map(models.Model):
         null=True, blank=True,
     )
 
+    # Vulnerability block
+    has_vulnerable_population = models.BooleanField(
```

Comment:
I was suffixing these with `_data` probably that's a bit pointless (eveything is data!) but maybe we should stick with that convention?